### PR TITLE
support user-supplied marker in list_blobs

### PR DESCRIPTION
**Pull request by larshagencognite**
Fixes https://github.com/Azure/azure-sdk-for-rust/issues/1465

### CODE REVIEW

File: sdk/storage_blobs/src/container/operations/list_blobs.rs
Comment (rbtcollins) [inline sdk/storage_blobs/src/container/operations/list_blobs.rs:26]:
```suggestion
    ?initial_marker: NextMarker,
```

Will make the code lower easier to read I think, because 'marker' is contextless

File: sdk/storage_blobs/src/container/operations/list_blobs.rs
Comment (rbtcollins) [inline sdk/storage_blobs/src/container/operations/list_blobs.rs:43]: if let Some(marker) = continuation.or(this.marker.as_ref()) {
    marker.append_to_url_query(&mut url);
}

File: sdk/storage_blobs/src/container/operations/list_blobs.rs
Comment (larshagencognite) [reply sdk/storage_blobs/src/container/operations/list_blobs.rs]: done

File: sdk/storage_blobs/src/container/operations/list_blobs.rs
Comment (demoray) [inline sdk/storage_blobs/src/container/operations/list_blobs.rs:26]:
```suggestion
    ?marker: NextMarker,
```

One of our primary goals is to align with other SDKs.  The golang SDK refers to this as "marker", rather than "initial_marker".

**Review by demoray — CHANGES_REQUESTED** [review]

File: sdk/storage_blobs/src/container/operations/list_blobs.rs
Comment (larshagencognite) [reply sdk/storage_blobs/src/container/operations/list_blobs.rs]: done

**Review by demoray — APPROVED** [review]

**Comment by larshagencognite** [comment]
> This looks good to me.
> 
> @larshagencognite in order to merge this, you need to agree to the contributor license agreement. Please see the message from [microsoft-github-policy-service](https://github.com/apps/microsoft-github-policy-service) listed above for instructions how to do this.

Great, just waiting for approval from my employer before signing.

**Review by cataggar — APPROVED** [review]
@larshagencognite, any luck getting approval? It would be nice to merge this before the next release.

**Comment by larshagencognite** [comment]
@cataggar All good now

**Pull request merged**
